ChangeLog.md

```diff
@@ -3,6 +3,308 @@
+- Increase fleet drop target width and add an icon to it
+- Modified how client and server communicate in a single player game
+to avoid firewall popups about network access
+- Added scanlines over fleets when not currently visible
+- Added nearly-accurate estimate of meter changes on the next turn to
+meter tooltips
+- Made MultiMeterStatusBar rescale to show largest value
+- In DesignWnd add ability to drag parts away from design
+
+
+#### Content
+
+- Translation updates: French, Russian
+- Removed / updated various missing, unused, or incorrect stringtable entries
+- Adjusted AI priorities for various buildings
+- Adjusted AI colonization planning
+- Made AI more cautious of hidden threats after losing fleets in a
+system it can't detect into
+- Made AI adjust priorities based on costs of ship parts and hulls
+- Adjusted AI priorities for research time
+- Adjusted AI industry modifiers
+- Adjusted AI weapon upgrade calculations
+- Updated pedia descriptions of various content and UI windows,
+including adding more links
+- Reworked intro sitrep to be less immersion-breaking
+- Moved start-of-game unlocking sitreps to before-the-first turn to
+avoid spamming the player
+- Removed "Average X" descriptions of species, leaving these only for
+the all-average humans
+- Made collapsing nebulae create various coloured stars
+- Added a "Rough Military Strength Estimate" empire statistic
+- Added Spinal Antimatter Cannon
+- Improved cross-platform consistency of random results in universe
+generation and in-game effects
+- Added Starlane Nexus and Starlane Bore buildings that can be used to
+create starlanes
+- Reworked Spatial Distortion Generator to push ships back along their
+approach starlane
+- Added Black Hole Collapser and Subspace Rift buildings
+- Adjusted some star names
+- Added sitreps to Planetary Starlane Drive effects
+- Various new hull art and slot positions, tech icons, building icons
+- Added the Accretion Disc special
+- Fixed / updated bioweapon sitreps
+- Added a random turn event that creates Space Krill in an empty system
+- Added new galaxy shape "Disc"
+- Added the feature to resurrect extinct species found in ancient
+ruins with the Xenoresurrection Lab
+- Added the new special Temporal Anomaly
+- Reworked weapon part refinements, so that instead of multiple parts,
+techs increase the strength of parts (although getting the increase to
+take effect requires being in supply range)
+- Added Energy Frigate hull, adjusted some other hulls' balance accordingly
+- Updated premade / default ship designs, made naming consistent
+- New bombardment weapons (feature in-progress)
+- Added EMP and electric overcharge ship parts
+- Added Solar Concentrators which increase laser weapon strength
+depending on local star type
+- Split up stealth part and planetary stealth techs
+- Added Flak Cannon ship part that attacks multiple times per combat
+turn but does minimal damage per attack
```

Comment:
Yes again. ^^

Actually, the Flak Cannon ship part is displayed in the Design Window (with Unavailable filter on), but there is no way to unlock it (no tech)